tests/checkbox.test.js

```diff
@@ -0,0 +1,62 @@
+import { JSDOM } from 'jsdom';
+import { isCompleted } from '../src/status';
+
```

Comment:
- [OPTIONAL] Good job done! It would be more meaningful to name this test file `status.test.js` so that it refers directly to the functional file.